spec/support/geckoboard_helpers.rb

```diff
@@ -81,12 +83,12 @@ def request_body_matches_geckoboard_data_for_claims?(request, claims)
         "policy" => claim.policy.to_s,
         "submitted_at" => claim.submitted_at&.strftime("%Y-%m-%dT%H:%M:%S%:z"),
         "passed_checking_deadline" => claim.check_deadline_date.past?.to_s,
-        "checked" => claim.check.present?.to_s,
+        "check" => claim.check.present? ? claim.check.result : "unchecked",
         "checked_at" => (claim.check.present? ? claim.check.created_at : DateTime.parse("1970-01-01")).strftime("%Y-%m-%dT%H:%M:%S%:z"),
-        "check_result" => claim.check.present? ? claim.check.result : "",
         "number_of_days_to_check" => claim.check&.number_of_days_since_claim_submitted,
         "paid" => claim.payment.present?.to_s,
         "paid_at" => (claim.payment.present? ? claim.scheduled_payment_date : DateTime.parse("1970-01-01")).strftime("%Y-%m-%dT%H:%M:%S%:z"),
+        "award_amount" => claim&.award_amount.to_i,
```

Comment:
By casting to an integer here we'd lose the decimal place.

Reply:
This is a limitation of [Geckoboard's money field](https://api-docs.geckoboard.com/#money-format).

Reply:
This is a limitation of [Geckoboard's money format](https://api-docs.geckoboard.com/#money-format).